lib/services/sqs_queue.rb

```diff
@@ -41,7 +41,13 @@ def notify_sqs(aws_access_key, aws_secret_key, payload)
     else
         queue = sqs.queues.named(queue_name)
     end
-    queue.send_message(clean_for_json(payload))
+    sqs.send_message(
+      queue_url: queue.url,
+      message_body: clean_for_json(payload),
+      message_attributes: {
```

Comment:
@bswinnerton, I just checked and i was using 1.66 locally which supports it but, 1.27.0 doesn't.  I've updated the code to use the aws-sdk-core (same as sns) and now it works correctly.